Dockerfile

```diff
@@ -17,7 +17,7 @@ COPY licenseCompleter.groovy /jenkins/src/licenseCompleter.groovy
 COPY show-pom-version.rb /jenkins/src/show-pom-version.rb
 
 WORKDIR /jenkins/src/
-RUN mvn clean install --batch-mode -Plight-test
+RUN ./mvnw clean install --batch-mode -Plight-test
```

Comment:
this seems wrong.  if we are using whatever version of maven then do not use a container with a specific version of maven embedded.
![image](https://user-images.githubusercontent.com/494726/96262667-03fa4880-0fba-11eb-89ee-951e5f946aed.png)

Reply:
Right, should change the base image here.